CareKit/Connect/OCKConnectTableViewCell.m

```diff
@@ -102,7 +103,12 @@ - (void)updateView {
     } else {
         _imageView.image = nil;
         _imageView.backgroundColor = OCKSystemGrayColor();
-        _monogramLabel.text = self.contact.monogram;
+        
+        NSString *monogram = self.contact.monogram;
+        NSRange stringRange = {0, MIN([monogram length], 2)};
+        stringRange = [monogram rangeOfComposedCharacterSequencesForRange:stringRange];
+        NSString *clippedMonogram = [monogram substringWithRange:stringRange];
+        _monogramLabel.text = clippedMonogram;
```

Comment:
@stevemoser Same idea, but with a slightly different approach.
```
NSString *monogram = self.contact.monogram;
NSString *clippedMonogram = [monogram substringToIndex:MIN(monogram.length, 2)];
```

We should also update this in `OCKConnectTableViewHeader.m`